logs_ingest/main.py

```diff
@@ -34,6 +34,7 @@
 record_age_limit = get_int_environment_value("DYNATRACE_LOG_INGEST_MAX_RECORD_AGE", 3600 * 24)
 attribute_value_length_limit = get_int_environment_value("DYNATRACE_LOG_INGEST_ATTRIBUTE_VALUE_MAX_LENGTH", 250)
 content_length_limit = get_int_environment_value("DYNATRACE_LOG_INGEST_CONTENT_MAX_LENGTH", 8192)
+cloud_log_forwarder = os.environ.get("WEBSITE_SITE_NAME", "")  # Function App name
```

Comment:
Does it provide full resource path? Including subscripiton, resource group, etc?

Reply:
No, it's only Function App name as we discussed before. $DEPLOYMENT_NAME + "-function"

Reply:
I think we had in mind a fully qualified function name (with the whole hierarchy embedded in the name and separated by slashes), right @dt-be ?

Reply:
Ahh, Beata is on a leave. But we definitely need a fully qualified resource name here. Otherwise you can have multiple functions with the same name in different subscriptions or resource groups and we wouldn't know which one is which. We avoid this ambiguity by using a fully qualified resource name